### ISSUE TITLE
Demo not working

### ISSUE THREAD

**Issue by asmith26** [modal-labs/llammer#13]
This sounds like a cool app, but unfortunately the demo doesn't seem to be working: https://modal-labs--quillman-web.modal.run/

Thanks!

**Comment by sanealytics** [comment]
Adding to this

```
modal run -q src.llm_vicuna --input "Where is the best sushi in New York?"
```

gets
```
 meson-python: error: Unsupported Python version 3.8.10, expected >=3.9
 ```
 
 I tried updating llm_vicuna.py to use 3.11 but still got this error.
 
 I don't have time to debug at the moment, but just reporting it.

**Comment by kning** [comment]
> This sounds like a cool app, but unfortunately the demo doesn't seem to be working: https://modal-labs--quillman-web.modal.run/
> 
> Thanks!

could you elaborate some more what's not working for you? seems to be working fine for me (though it did take some time to register my voice input)

https://github.com/modal-labs/quillman/assets/1820651/9bf10613-6fde-4f97-a47d-d1aa357c43f2

**Comment by asmith26** [comment]
Looks like this was fixed in #15, thanks!
